integration-tests/pom.xml

```diff
@@ -39,7 +39,7 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-junit5</artifactId>
-            <version>3.0.0.CR2</version>
+            <version>3.0.0.Final</version>
```

Comment:
You should remove the version here as it's handled by the BOM and it can be confusing.

Also any chance you could release a final release of JPAStreamer 3.0 before April 25th so that we have a nice release for the announcement. Thanks!